course-definition.yml

```
@@ -48,6 +48,15 @@ marketing:
         I think the instant feedback right there in the git push is really cool.
         Didn't even know that was possible!
 
+extensions:
+  - slug: "backreferences"
+    name: "Backreferences"
+    description_markdown: |
+      In this challenge extension, you'll add support for [backreferences][1] to your Grep implementation.
+      
+      Along the way, you'll learn about how to reuse earlier patterns in capturing groups.
```

```suggestion
      Along the way, you'll learn about how capture groups and backreferences work.
```

(Still think this can be refined a bit, but out of ideas for now)